python/delta/_version.py

```diff
@@ -0,0 +1,8 @@
+from pathlib import Path
+# delta.io version
+def get_version_from_sbt():
+    version = Path("../../version.sbt").read_text().strip()
```

Comment:
@nkarpov after I found a problem here. It won't work.

After creating python package version.sbt won't be available at specified. version.sbt or similar should be added python source path.